src/plugin/spine.js

```diff
@@ -374,16 +363,16 @@ pc.extend(pc, function () {
         for (i = 0; i < len; i++) {
             this._meshes[i].name = batches[i].mat;
             var mi = new pc.MeshInstance(this._node, this._meshes[i], this._materials[batches[i].mat]);
+            mi.drawOrder = i + (this.priority * 1000);
```

Comment:
I always felt that this was a hack, designed so that if you had two characters they would render one after the other, not sure if there is a better way to do this?

Reply:
Agree, at this point I have do this to enforce specific draw order of sub-meshes.